src/olympia/migrations/914-remove-addon-sitespecifc-flag.sql

```diff
@@ -0,0 +1 @@
+ALTER TABLE addons DROP COLUMN sitespecific;
```

Comment:
cc @jasonthomas that this is happening, should be relatively quick though and `addons` shouldn't be as big as `users` for example so I don't expect any problems here.